### PR TITLE
perf: remove quotes from slice

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Remove quotes from the slice, as we can access them directly from the cache.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

### Engineering

- Quotes should still work behind the multi hop flag
- Quote selection should work

### Operations

N/A

## Screenshots (if applicable)

N/A

### CODE REVIEW

**Comment by 0xApotheosis** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4860** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4860" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4860?utm_source=stack-comment).

File: src/state/apis/swappers/swappersApi.ts [hunk 1]
```diff
@@ -74,12 +74,12 @@ export const swappersApi = createApi({
         const orderedQuotes: ApiQuote[] = orderBy(
           quotesWithInputOutputRatios,
           ['inputOutputRatio', 'swapperName'],
-          ['asc', 'asc'],
+          ['desc', 'asc'],
         )
 
-        dispatch(tradeQuoteSlice.actions.setQuotes({ quotes: orderedQuotes, queryStartTime }))
+        dispatch(tradeQuoteSlice.actions.setSwapperName(orderedQuotes[0]?.swapperName))
```
Comment (0xApotheosis) [inline src/state/apis/swappers/swappersApi.ts:80]: We still need to decide how we handle swapper selection resets on new quotes. Leaving as-is behaviour for now.

File: src/state/apis/swappers/swappersApi.ts [hunk 2]
```diff
@@ -18,10 +18,10 @@ import { BASE_RTK_CREATE_API_CONFIG } from '../const'
 export const swappersApi = createApi({
   ...BASE_RTK_CREATE_API_CONFIG,
   reducerPath: 'swappersApi',
+  keepUnusedDataFor: 60, // this needs to be higher than the polling interval of any consumers
```
Comment (woodenfurniture) [inline src/state/apis/swappers/swappersApi.ts:21]: ❤️

**Review by woodenfurniture — APPROVED** [review]
good stuff

**Pull request merged**